api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -799,32 +799,40 @@ public CohortDefinition getNumberOfPatientTbScreenedInFirstEncounter() {
-            + "                              AND e.voided = 0 "
-            + "                              AND p.voided = 0"
-            + "  						AND o.voided = 0 "
-            + "                      AND o.concept_id   = ${tbSymptomsConcept}"
-            + "                      AND o.value_coded  IN (${yesConcept}, ${noConcept})";
+        "SELECT form.patient_id FROM "
+            + "(SELECT p.patient_id,e.encounter_id "
+            + "                                 FROM patient p "
+            + "                                 INNER JOIN encounter e "
+            + "                                 ON p.patient_id = e.patient_id "
+            + "                                 INNER JOIN obs o "
+            + "                   ON o.encounter_id = e.encounter_id "
+            + "                 WHERE e.encounter_type = ${adultoSeguimentoEncounterType} "
+            + "                                 AND e.location_id = :location "
+            + "                                 AND e.voided = 0 "
+            + "                                 AND p.voided = 0 "
+            + "                 AND o.voided = 0 "
+            + "                         AND o.concept_id  = ${tbSymptomsConcept} "
+            + "                         AND o.value_coded IN (${yesConcept},${noConcept}) "
+            + "                         AND e.encounter_datetime BETWEEN :startDate AND :endDate) AS form "
+            + "                                 RIGHT JOIN "
+            + "                                 (SELECT minDate.patient_id, encounter_id, encounter_datetime,min_date FROM  "
+            + "  (SELECT p.patient_id,MIN(e.encounter_datetime) AS min_date "
+            + "               FROM encounter e "
+            + "                                   INNER JOIN patient p on p.patient_id = e.patient_id "
+            + "                                   INNER JOIN obs o "
+            + "                                     ON o.encounter_id = e.encounter_id "
+            + "                               WHERE e.encounter_type = ${adultoSeguimentoEncounterType} "
+            + "                                 AND e.location_id = :location "
+            + "                                 AND e.voided = 0 "
+            + "                                 AND p.voided = 0 "
+            + "                                  AND o.voided = 0 "
+            + "                            AND e.encounter_datetime BETWEEN :startDate AND :endDate  "
+            + "                           GROUP BY p.patient_id) minDate "
+            + "                          INNER JOIN encounter enc ON minDate.patient_id = enc.patient_id  "
+            + "                          WHERE minDate.min_date = enc.encounter_datetime) AS encounters "
+            + "                         ON encounters.encounter_id = form.encounter_id "
+            + "                         WHERE form.patient_id is not null"
```

Comment:
Make this **is not null**  to **IS OT NULL** like everywhere else you have considered key and preserved words

Reply:
Done.

Reply:
Thanks